Documentation/iOS/MobileCenter/.jazzy.yaml

```diff
@@ -5,7 +5,7 @@ sdk: iphonesimulator
 theme: ../../Themes/apple
 
 module: MobileCenter
-module_version: 0.11.0
+module_version: 0.11.1
```

Comment:
Keep in mind that you also need to change the version in other yaml files for other platforms.